man/systemd.special.xml

```diff
@@ -181,8 +182,8 @@
         <varlistentry>
           <term><filename>cryptsetup.target</filename></term>
           <listitem>
-            <para>A target that pulls in setup services for all
-            encrypted block devices.</para>
+            <para>A target that pulls in setup services for all encrypted block devices.
+            See <filename>initrd-cryptsetup.target</filename> for the initramfs equivalent.</para>
```

Comment:
when you touch it, better also add reference to remote-cryptsetup.target, and replace "all encrypted" by "local encrypted" or so. ie. maybe:

> A target that pulls in setup services for all local encrypted block devices. See <filename>initrd-cryptsetup.target</filename> below for the equivalent target for the initrd, and <filename>remote-cryptsetup.target</filename> below for the equivalent for remote volumes.

or something like that